OrbitGl/App.cpp

```diff
@@ -607,14 +615,19 @@ void OrbitApp::ClearCapture() {
   AddTopDownView(*empty_sampling_profiler);
   Capture::GSamplingProfiler = empty_sampling_profiler;
 
+  if (selection_report_) {
+    auto empty_selection_profiler = std::make_shared<SamplingProfiler>(Capture::GTargetProcess);
```

Comment:
If #1077 will be in before this change, it will silently lead to compilation error, as GTargetProcess will be gone. Otherwise, I'll take care of updating #1077.

Reply:
I can wait for your PR and rebase then